packages/graph-editor/src/components/panels/dropPanel/dropPanel.tsx

```diff
@@ -138,8 +138,12 @@ export const DropPanelInner = observer(({ data }: IDropPanel) => {
                     width="full"
                     css={{ padding: '$3 0' }}
                   >
-                    <IconoirProvider
-                      iconProps={{ width: '0.875em', height: '0.875em' }}
+                    <div
```

Comment:
again you can remove it here

Reply:
if you ever need to change the size of an icon, you can pass in `width={32} height={32}` or so. by default icons have a size of 16.

if you change it to e.g. 32 we adjust the stroke width automatically

Reply:
I can't use em in image size directly though, so the em size is not needed?